app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -42,9 +43,22 @@ def create
         end
       end
 
+      def update
+        super
+        if params[:content_access_mode] == 'simple_content_access'
+          async_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:id])
+          info "Enabling Simple Content Access for organization #{@taxonomy.name}.", link: { text: "View progress on the Tasks page", href: "/foreman_tasks/tasks" }
```

Comment:
Users may enable/disable SCA for an organization that is not `Organization.current`.  Also, the Org details page is run with organization unscoped, so we don't even know what org is selected anyway.  For these reasons, it makes more sense to refer users to the Tasks page rather than the Manage Manifest modal.